build-sensor/include/step3.h

```diff
@@ -0,0 +1,34 @@
+class step3 {
+
+    public:
+        void setup() {
+
+            // Configure Serial Port for monitoring
+            Serial.begin(115200);
+
+            // Configure mode of LED port
+            pinMode(LED_BUILTIN, OUTPUT);
+
+            // Start the WiFi connection
+            WiFi.enableSTA(true);
+            WiFi.begin(WIFI_SSID, WIFI_PASS);
+
+            // Check the status of Wifi before we continue
+            while (WiFi.status() != WL_CONNECTED) {
+                // 1 second delay between checking status
+                delay(1000);
+                Serial.printf("New Client. RSSi: %i dBm\n", WiFi.RSSI());
```

Comment:
If we are not yet connected to an access point what does "New Client" refer to?

Reply:
True, it is displaying the receive signal strength, I've updated to better describe what it is doing.